bootstrap/handlers/httpserver.go

```diff
@@ -123,7 +125,6 @@ func (b *HttpServer) BootstrapHandler(
 		defer wg.Done()
 
 		<-ctx.Done()
-		lc.Info("Web server shutting down")
```

Comment:
why remove this?

Reply:
Felt it was redundant with the `lc.Info("Web server shut down")` message 2 lines down.